http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/FileUploadClientResource.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import io.quarkus.ts.http.restclient.reactive.multipart.FileUploadClient;
+import io.quarkus.ts.http.restclient.reactive.multipart.MultipartBodyWithMediaType;
+import io.quarkus.ts.http.restclient.reactive.multipart.MultipartBodyWithoutMediaType;
+
+@Path("/test/upload")
```

Comment:
Can you change this to something like `/client/upload` as for me it's not clear why is the `test` part there.